devito/ops/__init__.py

```diff
@@ -17,6 +19,7 @@
                   'speculative': PlatformRewriter})
 
 # The following used by backends.backendSelector
+from devito.ops.compiler import CompilerOPS as Compiler # noqa
```

Comment:
I don't really like that, this should be another compiler in `devito.compiler` not a different one here, especially considering there is a lot of copy-paste from devito compiler.

Reply:
Our original idea was defining everything that would work differently for ops inside the ops folder. I thought that the compilerOPS case would be similar to the operatorOPS, and tried to make them work alike. As we work further on the compiler, there won't be as much copy-paste

Reply:
If you look at how Yask is setup for example, the backend selector takes care of the selection but everything is still called the same and the compiler goes in `compiler.py`.  The same way `OperatorOPS` shouldn't exist the backend selector should choose ops depending on DEVITO_BACCKEND but the call should still be `Operator(..)` no matter what  not `OperatorOPS`

Reply:
I'm not quite sure I follow... From what I saw from Yask, it declares its own compiler inside the __init__.py 

https://github.com/opesci/devito/blob/cfb3d380690ea10d05498024f365d51b68cc4a16/devito/yask/__init__.py#L70